catalog/ui/search-ui-app/src/main/resources/etc/IntrigueBundle.properties

```diff
@@ -4,4 +4,6 @@ sources.title=Sources
 sources.options.all=All Sources
 sources.polling.error.title=Error Polling Sources
 sources.polling.error.message=Unable to query server for list of active sources
-search.sources.selected.none.message=No sources are currently selected. Edit the search and select at least one source.
\ No newline at end of file
+search.sources.selected.none.message=No sources are currently selected. Edit the search and select at least one source.
+form.title=Form
+forms.title=Forms
```

Comment:
consider moving these props out or renaming this file to `US-en.properties`. It may be worth validating these files at compile time to verify all entries are present etc.